packages/generator/src/generator.ts

```diff
@@ -75,12 +79,44 @@ export async function generate(options: GeneratorOptions): Promise<void> {
       directories,
       options.processesJsGeneration || defaultValueProcessesJsGeneration
     );
-    await chunks.reduce(
-      (all, chunk) => all.then(() => transpileDirectories(chunk)),
-      Promise.resolve()
-    );
+    try {
+      await chunks.reduce(
+        (all, chunk) => all.then(() => transpileDirectories(chunk)),
+        Promise.resolve()
+      );
+    } catch (err) {
+      if (err.message?.includes('error TS2307')) {
```

Comment:
I would like to have a hidden boolean option `failOnDependency` to enable or disable this. In our api hub pipeline it is convinient to see a failing transpilation. Also for other E2E test this could become in handy

Reply:
This will fail either way. It just adds another log message.